HCCGo/app/html/beta_notice.html

```diff
@@ -0,0 +1,10 @@
+<p>Welcome to the HCCGo Beta.  With HCCGo, you are able to submit jobs, transfer files, and monitor your usage of HCC resources</p>
```

Comment:
whoops, didn't realize I accidentally added it in the oldest commit, will do that